src/iedit.cpp

```diff
@@ -2370,6 +2414,26 @@ void iedit_parse(struct descriptor_data * d, const char *arg)
             case CYB_REFLEXTRIGGER:
             case CYB_SKULL:
             case CYB_TORSO:
+            case CYB_STRENGTHMOD:
+            case CYB_QUICKNESSMOD:
+            case CYB_ARMORMOD:
+            case CYB_CYBERARMGYROMOUNT:
+            case CYB_CYBERARMS:
+              if (number < 0 || number > NUM_ARMSMODS+1) {
+                send_to_char("Invalid Input! Enter options (0 to quit): ", CH);
+                return;
```

Comment:
Recommend using the logic below for CYB_HANDRAZOR so you set the bits like you're intending to do. Currently, this will take the exact value put in and use it for the value instead of setting the bit like you want it to.